src/component/typingbox/__tests__/TypingBox.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import TypingBox from '../TypingBox';
+import renderer from 'react-test-renderer';
+import { mount, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+describe("TypingBox", () => {
+  const onChange = jest.fn();
+  const start = jest.fn();
+  const onWordComplete = jest.fn().mockReturnValue(false);
+
+  const buildProps = (newProps = {}) => ({
+    value: "",
+    onChange,
+    onWordComplete,
+    start,
+    ...newProps,
+  });
+
+  it("renders an input field", () => {
+    const TypingBoxComponent = renderer.create(<TypingBox {...buildProps()}/>).toJSON();
+    expect(TypingBoxComponent).toMatchSnapshot();
+  });
+
+  it("renders the input field as empty when we pass an empty value", () => {
+    const TypingBoxComponent = mount(<TypingBox {...buildProps()}/>)
+    const value = TypingBoxComponent.find('input').props().value;
```

Comment:
Since we already have a prop called `value`, I might elect to call this `inputValue` to clarify that it is different than our prop.